scripts/config.js

```diff
@@ -0,0 +1,3 @@
+var sort_options = ["season", "mix_type", "location"];
+var json_url = "https://s3-us-west-1.amazonaws.com/lctk.data/derin-test/2_plots_list.json";
+var filters = ["normalization"];
```

Comment:
Don't put these variables in a separate files, they should be in the file they get used in. It causes confusion and people may not be aware where it get loaded. Plus the skews the scope of them.

Reply:
fixed